### ISSUE TITLE
Restored domain and PHP error log not enabled

### ISSUE THREAD

**Issue by iliaross** [virtualmin/virtualmin-gpl#531]
The restored domain adds PHP log entry in Log File Rotation.

<img width="533" alt="image" src="https://user-images.githubusercontent.com/4426533/224515089-5a5def03-24ec-4054-a949-a33fca96f37c.png">

However, actual PHP error log for the domain happens to be disabled:

<img width="662" alt="image" src="https://user-images.githubusercontent.com/4426533/224515082-8ff56dfb-3bd6-4dd9-ac3c-7f3cfe92b733.png">

The correct is to have actual PHP error log enabled for the domain, because it was enabled on the source system (where backup was created).

**Comment by jcameron** [comment]
That's odd - was this a restore of a domain already on the system, or a domain that was re-created as part of the restore process?

**Comment by iliajie** [comment]
It was a restore of non-existing domain.

**Comment by jcameron** [comment]
Was this domain using FPM or fCGId PHP execution mode? And was the error_log set in the domain's FPM pool file or php.ini ?

**Comment by iliajie** [comment]
It was FPM, and no actual `error_log` directive wasn't set.

**Comment by iliajie** [comment]
It was FPM, and no actual `error_log` directive ~wasn't~ was set in the pool file.

**Comment by jcameron** [comment]
Ok it looks like we don't save the error_log setting at all ... I'll fix that.

**Comment by iliajie** [comment]
It does save it on the domain config file now, but it still has no effect on the PHP-FPM pool config file, i.e. actual `error_log` directive isn't really set.

**Comment by iliajie** [comment]
Yes, I'm pretty sure.

**Comment by jcameron** [comment]
Does this also happen for a domain using fcgi to execute PHP scripts?

**Comment by iliajie** [comment]
That I'm not sure of, and would have to test it.

**Comment by iliajie** [comment]
> Does this also happen for a domain using fcgi to execute PHP scripts?

Yes, just triple-checked. It does also happen in case of FCGI or CGI, and `error_log` is **_not_** being being set. Also (not sure if it's expected) even though target system has all PHP execution modes available, yet upon restore execution mode is not preserved. E.g. if source has FCGI set upon backup, when restored on the target it defaults to whatever is set on the template (FPM).

**Comment by jcameron** [comment]
Can you post a series of Virtualmin API commands that would demonstrate this problem, by creating a domain, backing it up and restoring it? I was unable to reproduce it in my tests, sorry..

**Comment by iliajie** [comment]
I did this using UI. Just try to restore [this](https://github.com/virtualmin/virtualmin-gpl/files/11009451/ubuntu22-pro.local-ssl-sharing-enabled.tar.gz) backup and see how it goes.

On the template I have defaults on:

<img width="315" alt="image" src="https://user-images.githubusercontent.com/4426533/226134127-0641a1cb-baf7-427e-aaa0-eb8c881aabe9.png">

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.
